saritasa_s3_tools/django/drf_fields.py

```diff
@@ -81,10 +82,28 @@ class S3UploadURLField(serializers.CharField):
 
     def __init__(self, **kwargs) -> None:
         """Make custom initialization."""
+        # Remove explicit max_length limit in field
+        # It causes issues with auto spec generation and validation.
+        # Since it can return full urls with auth query, which can easily pass
+        # a limit specified in model field(which is by default 100), it causes
+        # confusion for openapi specs validators.
+        max_length = kwargs.pop("max_length", None)
         super().__init__(**kwargs)
+        if max_length is not None:
```

Comment:
Dont we need test for it or it already has?

Reply:
It's copied from original implementation(drf's char field). I'm planning to setup coverage reporting similar to https://github.com/saritasa-nest/django-import-export-extensions